### DISCUSSION TITLE
date input and widgets

### DISCUSSION THREAD

**Discussion by basicx-shivani** [atoti/atoti#536] [General]
firstly I need start and end date from users and then table will explode  by that dates and dashboard will created.
Now I am facing issue with input widgets.

**Comment by bloussou** [comment]
Hello,
I'm not sure to understand what you mean. What is an `input widget` ? What do you mean by `explode` ? Should the dashboard have been created automatically based on dates specified by users ?

**Reply by basicx-shivani** [reply]
yes, I want dashboard created according to users date input or date range.

**Reply by bloussou** [reply]
You can set filters on dashboard, page or widget :
![image](https://user-images.githubusercontent.com/32448554/159869353-f7044dfb-a24f-476c-947e-5b55f14ae67e.png)
You can also use the quick filter widget (ti will apply filters on the page):
![image](https://user-images.githubusercontent.com/32448554/159869545-bfa0d2bf-c08d-456c-a545-1a76ed2bffb5.png)

Is it what you are looking for ?
